commands/FBDebugCommands.py

```diff
@@ -189,6 +191,86 @@ def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
 
 
+def switchBreakpointState(expression,on):
+  ci = lldb.debugger.GetCommandInterpreter()
+  res = lldb.SBCommandReturnObject()
+
+  expression_pattern = re.compile(r'{}'.format(expression),re.I)
+
+  target = lldb.debugger.GetSelectedTarget()
+  for breakpoint in target.breakpoint_iter():
+    for location in breakpoint:
+      if expression_pattern.search('{}'.format(location)):
+        print location
+        location.SetEnabled(on)
+
+class FBMethodBreakpointEnableCommand(fb.FBCommand):
+  def name(self):
+    return 'rbenable'
+
+  def description(self):
+    return """
+    Enable a set of breakpoints for a regular expression
+
+    Examples:
+
+          #use `rbenable disabled` to switch all breakpoints to `enable`
+          benable disabled
```

Comment:
This doesn't appear to be implemented. I don't think it's necessary to implement, since `br dis` (`breakpoint disable`) does this already.

Reply:
since `br dis` (`breakpoint disable`)has already this.
Should we use `location.GetAddress()` to search?

Reply:
![image](https://user-images.githubusercontent.com/7285119/36824567-0281d3f4-1d3e-11e8-8baf-1e7eb1cb2a2a.png)

```
location	57.1: where = libsystem_kernel.dylib`mach_msg_trap + 4, address = 0x00000001832ef564, resolved, hit count = 0 
address		libsystem_kernel.dylib`mach_msg_trap + 4
symbol		id = {0x0000024e}, range = [0x000000018095b560-0x000000018095b56c), name="mach_msg_trap"
name		mach_msg_trap
```

when i set breakpoint at this address(`0x0000024e`),only `symbol` contain `0x0000024e`.
So i want search `address` and `symbol`.How do you think?

Reply:
I don't know what `id = {0x0000024e}` represents? The address of the breakpoint is in the `location`: `address = 0x00000001832ef564`.

Reply:
i am sorry.i mean that i want search `0x1832ef564` ,not `0x00000001832ef564`

Reply:
Do you mean you want `rbdisable 0x1832ef564` to disable the breakpoint whose address is `0x00000001832ef564`?

Reply:
yes

Reply:
Maybe have add special cased input handling. If the user's input expression matches `0x[[:xdigit:]]`, then when building the regex, add a `0*` after the `0x`. So `0x1832ef564` would become `0x0*1832ef564`.